### ISSUE TITLE
Can't reproduce the result in the paper

### ISSUE THREAD

**Issue by Wxinyao** [charlio23/DeepSkeleton-pytorch#2]
Hi! I am trying to use your code to train the model, but the loss stays very even after dozens of epochs and I don't know why. Have you successfully reproduce the result in the paper? If yes, would you please upload the pretrained model. Thanks!
